core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -98,16 +91,16 @@ angular.module('oppia').directive('aboutPage', [
             WindowRef.nativeWindow.onhashchange = function() {
               const hashChange = window.location.hash.slice(1);
               if (hashChange === ctrl.TAB_ID_FOUNDATION || (
-                hashChange === 'license')) {
-                activateTab(ctrl.TAB_ID_FOUNDATION);
+                hashChange === ctrl.TAB_ID_LICENSE)) {
+                ctrl.activeTabName = ctrl.TAB_ID_FOUNDATION;
                 // Ensure page goes to the license section
-                if (hashChange === 'license') {
+                if (hashChange === ctrl.TAB_ID_LICENSE) {
                   WindowRef.nativeWindow.location.reload(true);
                 }
               } else if (hashChange === ctrl.TAB_ID_CREDITS) {
-                activateTab(ctrl.TAB_ID_CREDITS);
+                ctrl.activeTabName = hashChange;
               } else if (hashChange === ctrl.TAB_ID_ABOUT) {
-                activateTab(ctrl.TAB_ID_ABOUT);
+                ctrl.activeTabName = hashChange;
```

Comment:
There are still unnecessary ifs... This can be simplified to:
```typescript
if (hashChange === ctrl.TAB_ID_LICENSE) {
  ctrl.activeTabName = ctrl.TAB_ID_FOUNDATION;
  WindowRef.nativeWindow.location.reload(true);
} else {
  ctrl.activeTabName = hash;
}
```

Reply:
Done.